client/settings/support-phone-input/index.js

```diff
@@ -41,6 +50,13 @@ const SupportPhoneInput = ( { setInputVallid } ) => {
 		}
 	}, [ supportPhoneError, setInputVallid ] );
 
+	let labelText = __( 'Support phone number', 'woocommerce-payments' );
+	if ( isDevModeEnabled ) {
+		labelText += __(
+			' ( Use +1 000-000-0000 in test mode. )',
```

Comment:
Nitpick but two first points would be required to change. Two latter points are up to your judgement.  

1. `test` -> `dev` 
2. There would be no space after `(` and before `)`. (Quickly putting this phrase in LanguageTools.org shows this suggestion). 
3. I don't think we need to have a period at the end. 
4. May soften the language a bit 😬 Otherwise, it feels like we're giving an order to merchants. Then, I suggest using the passive voice here. 
```suggestion
			' (+1 000-000-0000 can be used in dev mode)',
```

Reply:
done